mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -9,16 +9,19 @@ CLASS avx net/minecraft/entity/player/PlayerEntity
 		METHOD a getText ()Ljm;
 	FIELD bA container Layb;
 	FIELD bB hungerManager Laxx;
+	FIELD bF experiencePickUpDelay I
 	FIELD b SIZES Ljava/util/Map;
 	FIELD bM isInWater Z
 	FIELD bN abilities Lavu;
-	FIELD bO experience I
-	FIELD bP experienceLevel I
-	FIELD bQ experienceLevelProgress F
+	FIELD bO xpLevel I
+	FIELD bP xpTotal I
+	FIELD bQ xpProgress F
 	FIELD bR enchantmentTableSeed I
 	FIELD bT fishHook Latj;
+	FIELD bU lastLeveledUp I
```

Comment:
What does it do? Should it be `lastLevel`?

Reply:
`lastPlayedLevelUpSoundTime` is more accurate actually. It's set in `addExperienceLevels` to the current entity's age and used to avoid a sound spam when players level up rapidly.